openshift/scripts/oc_provision_crunchy_standby.sh

```diff
@@ -35,7 +38,8 @@ OC_PROCESS="oc -n ${PROJ_TARGET} process -f ${TEMPLATE_PATH}/crunchy_standby.yam
 -p SUFFIX=${SUFFIX} \
 -p TARGET_NAMESPACE=${PROJ_TARGET} \
 -p BUCKET=${BUCKET} \
--p DATA_SIZE=45Gi \
+-p DATE=${DATE} \
+-p DATA_SIZE=65Gi \
```

Comment:
Could we parameterize `DATA_SIZE` and `WAL_SIZE`? It will give us a little more flexibility around test deployments.